File: b/soccerTeam.java
public class soccerTeam
{
	private String name;
	private String nation;
	private int year_founded;
	private String[] player;

	public soccerTeam(String name, String nation, int year_founded, String[] player)
	{
		this.name = name;
		this.nation = nation;
		this.year_founded = year_founded;
		this.player = player;
	}

	public soccerTeam(){}

	public void setName(String name)
	{
		this.name = name;
	}

	public String getName()
	{
		return name;
	}

	public void setNation(String nation)
	{
		this.nation = nation;
	}

	public String getNation()
	{
		return nation;
	}

	public void setYear_founded(int year_founded)
	{
		this.year_founded = year_founded;
	}

	public int getYear_founded()
	{
		return year_founded;
	}

	public void setPlayer(String[] player)
	{
		this.player = player;
	}

	public String[] getPlayer()
	{
		return player;
	}

	public void printTeam()
	{
		System.out.println("\nNama tim : " + name);
		System.out.println("Negara asal tim : " + nation);
		System.out.println("Tahun berdiri : " + year_founded);
		System.out.println("Pemain :");
		for(String i: player)
		{
			System.out.println(" -> " + i);
		}
	}
}

File: b/soccerTeam.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

class soccerTeam
{
	private:
		string name;
		string nation;
		int year_founded;
		vector<string> player;

	public:
		// construct
		soccerTeam(string name, string nation, int year_founded, vector<string> player)
		{
			this->name = name;
			this->nation = nation;
			this->year_founded = year_founded;
			this->player = player;
		}

		// default construct
		soccerTeam(){}

		void setName(string name)
		{
			this->name = name;
		}

		string getName()
		{
			return name;
		}

		void setNation(string nation)
		{
			this->nation = nation;
		}

		string getNation()
		{
			return this->nation;
		}

		void setYear_founded(int Year_founded)
		{
			this->year_founded = year_founded;
		}

		int getYear_founded()
		{
			return this->year_founded;
		}

		void setPlayer(vector<string> player)
		{
			this->player = player;
		}

		vector<string> getPlayer()
		{
			return this->player;
		}

		// deconstruct
		~soccerTeam(){}

		void printTeam()
		{
			cout << "\nNama tim : " << this->name << endl;
			cout << "Negara asal tim : " << this->nation << endl;
			cout << "Tahun berdiri : " << this->year_founded << endl;
			cout << "Pemain :" << endl;
			for(string i: player)
			{
				cout << " -> " << i << endl;
			} 
		}
};